src/components/appNavigation/TabsView.js

```diff
@@ -76,16 +75,11 @@ const RewardButton = React.memo(({ onPress, style }) => {
   const [updatesCount, setUpdatesCount] = useState(0)
 
   useEffect(() => {
-    const updateIcon = async () => {
-      const lastState = (await AsyncStorage.getItem('GD_lastInviteState')) ||
-        userStorage.userProperties.get('lastInviteState') || { pending: 0, approved: 0, total: 0 }
+    const lastState = userStorage.userProperties.get('lastInviteState') || { pending: 0, approved: 0, totalEarned: 0 }
```

Comment:
using a **single** source of trust to avoid collisions.
also lastInviteState **wasn't** actually cleared on logout. but add code to removed it - it's **one more workaround**
i prefer to use the correct way and use just single source of trust.
in addition, user props **also** cached in async storage so storing GD_lastInviteState makes **no sense** at all